repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/mocks/MockValkyrie.groovy

```diff
@@ -76,10 +76,12 @@ class MockValkyrie {
     String account_perm = "also_butts"
     String contact_id = ""
     String tenant_id = ""
+    Integer fixnumber = 10
+    Integer multiplier = 1
 
```

Comment:
yes, that why I don't want to slow other tests just set fixnumber as 10 and use multiplier as flexible number and use only as inventory call is made